### ISSUE TITLE
cli creates .wakatime-project in root

### ISSUE THREAD

**Issue by weeebdev** [wakatime/wakatime-cli#575]
Cli creates `.wakatime-project` in the root directory (`~/` in my case) which results in all of my projects being pushed tagged with the generated name provided in `~/.wakatime-project`.

**Expected behavior (what should have happened)**:
Create `.wakatime-project` only in specified paths in `hide_project_names` flag
**Actual behavior (what went wrong)**:
It works but also generates a project name in root.
**Environment**:
Ubuntu 20.04.3 LTS
wakatime version `13.1.0`, python I guess

**Logs**:
Secret info but for the date when `~/.wakatime-project` file was created there are only errors about `failed to run command: sending heartbeat(s) later due to api error: won't send heartbeat due to backoff`

My config
```cfg
[settings]
api_key = {secret}
status_bar_coding_activity = true
status_bar_enabled = true
hide_file_names = 
    ^/home/nickname/git/private
    ^/home/nickname/job
hide_project_names = 
    ^/home/nickname/git/private
    ^/home/nickname/job
```

**Comment by weeebdev** [comment]
The workaround I found last month is to create an empty `~/.wakatime-project` file, but for some reason unknown to me it regenerated the file at the beginning of this month

**Comment by gandarez** [comment]
I guess the projects you've been working don't have any of these (`git`, `hg`, `tfs`, `svn`) source controls attached to them and at root folder you have one of them. Could you confirm it?

**Comment by gandarez** [comment]
@weeebdev any update?

**Comment by weeebdev** [comment]
> @weeebdev any update?

Oh, sorry. Been busy with my job. So, the projects I was working with do have `.git` most of the time. But `^/home/nickname/git/private` `^/home/nickname/job` paths don't. There I keep projects which I want to keep obfuscated

**Comment by weeebdev** [comment]
Let me make it clear. I want my projects under `^/home/nickname/git/private` `^/home/nickname/job` paths to be obfuscated, but not the other ones. But, for some reason I don't know, cli creates `.wakatime-project` file with random generated name at the home directory (`~`) at a random period of time, which ends up all of my projects being tagged with this random name.

**Comment by alanhamlett** [comment]
@weeebdev is this fixed for you after #598?

**Comment by weeebdev** [comment]
> @weeebdev is this fixed for you after #598?

I don't know, as I said, it appears randomly. I'll update and wait. For now, I think it's okay to close the issue
